frontend/appflowy_flutter/lib/plugins/database_view/widgets/card/container/accessory.dart

```diff
@@ -66,7 +66,8 @@ class CardAccessoryContainer extends StatelessWidget {
     return Container(
       clipBehavior: Clip.hardEdge,
       decoration: decoration,
-      child: child,
+      child: ClipRRect(
+      borderRadius: const BorderRadius.all(Radius.circular(4)),child: child),
```

Comment:
Hey there @onyicodes. There is a flutter analyzer warning with the trailing commas. Could you take a look? Apart from that, this looks good to me.

Reply:
Thanks @richardshiue, I've fixed that